src/pheval/config_parser.py

```diff
@@ -0,0 +1,24 @@
+from dataclasses import dataclass
+from pathlib import Path
+
+import yaml
+from serde import serde
+from serde.yaml import from_yaml
+
+
+@serde
+@dataclass
+class InputDirConfig:
+    tool: str
+    software_directory: Path or None
```

Comment:
Is this possibley redundant? Isnt this the same as "the directory the software_config is in"?

Reply:
Yes, it could be redundant, I just wasn't sure how the input directory would be laid out, whether for example the application.properties for Exomiser would be put just in the input directory:

```
└── exomiser-13.2.0
    └── default
        ├── 2302_hg19
        ├── 2302_phenotype
        ├── application.properties
        └── exomiser-cli-13.2.0
```

or within the exomiser-cli directory

```

└── exomiser-13.2.0
    └── default
        ├── 2302_hg19
        ├── 2302_phenotype
        └── exomiser-cli-13.2.0
            └── application.properties
```

Reply:
What is the advantage to give structure, rather then simply a flat list (first example)?

Reply:
Sure, it does give it structure, if the location of the software config was in the software directory, it makes sense, and as you said it can allow for one less field in the config. Whatever you feel works better structure wise I'm happy to go with and make the changes

Reply:
removed this field within the input dir config